pkg/sysregistriesv2/system_registries_v2.go

```diff
@@ -665,6 +681,16 @@ func (c *tomlConfig) loadConfig(path string, forceV2 bool) error {
 		c.V2RegistriesConf = *v2
 	}
 
+	// Merge the alias maps.  New configs override previous entries.
+	newAliases := c.Aliases
+	c.Aliases = prevAliases // point back to the previous map and override
+	if c.Aliases == nil {
+		c.Aliases = make(map[string]string)
+	}
+	for name, value := range newAliases {
+		c.Aliases[name] = value
+	}
+
 	// Post process registries, set the correct prefixes, sanity checks, etc.
 	if err := c.postProcess(); err != nil {
```

Comment:
Should aliases be validated for some format properties? Or maybe only in `Resolve`…?

Reply:
If we want to validate I vote for validating here. If users have a malconfigured alias, I want them to know early on (ideally at start up). Otherwise it's entirely unpredictable when and if a user will hit the error when resolving to a malconfigured alias.

Reply:
Aliases are now validated on load and on `AddAlias`.